docs/index.mdx

```diff
@@ -2,14 +2,42 @@
 sidebar_position: 1
 ---
 
-# Learning Green Software
+# Welcome
 
-This learning site contains materials created by the [Community Working Group](https://community.greensoftware.foundation) in The [Green Software Foundation](https://greensoftware.foundation) to educate software practitioners in topics related to green software.
+Do you want to be a part of the green software movement?
 
-This site exists because the concepts a software practitioner needs to know to build, maintain and run greener applications are not taught in traditional software curriculums. The lack of a shared language or standardized terms complicates green software communication.
+If you want to make your software applications more sustainable and build a career in green software, then this training is for you. It will teach you how to build, maintain and run greener applications irrespective of the application domain, industry, organization size or type, programming language, or framework.
 
-As organizations look to hire green software talent, we need a method of evaluating the knowledge of software practitioners to ensure they meet a minimum and consistent level of understanding.
+If you pass the associated exam, it comes with a certificate of completion from the Linux Foundation.
 
-This site contains the training for various Green Software courses including the [Green Software Practitioner Course](https://learn.greensoftware.foundation/practitioner/introduction) which is the first training we recommend anyone to take.
+## What is a software practitioner?
 
-The main website for this material is https://learn.greensoftware.foundation and the source for this site can be found on our GitHub https://github.com/Green-Software-Foundation/training.
+A software practitioner is anyone who is involved in the practice of building, deploying, or managing a software application. This includes but is not limited to the following:
+
+- Developers
+- Dev Ops
+- Testers
+- Architects
+- SREs
+- Product Managers
+- UX Designers
+
+## How do you get the best out of this course?
+
+- Have at least **one year of industry experience** building, deploying, or managing software applications.
+- Complete each section in order from start to finish.
+- Complete the knowledge checks at the end of each section.
+- If you would like to gain a certificate of completion, take the exam through the Linux Foundation.
+
+## By the end of this training, you will understand the following:
+
+- The core drivers of carbon emissions for software applications.
+- The 3 actions you can take to reduce the carbon emissions of your software.
+- How to measure software emissions using the Greenhouse Gas (GHG) or the Software Carbon Intensity (SCI) protocols.
```

Comment:
Just checking that we also want to use the word "protocols" for SCI, not specification?

Reply:
Good catch, updated :)